Dockerfile

```diff
@@ -11,10 +11,10 @@ RUN npm install --include=dev
 COPY . .
 
 ARG ARG_PUBLIC_URL
-ARG ARG_REACT_APP_BACKEND_URL
+ARG ARG_VITE_BACKEND_URL
 
 ENV PUBLIC_URL=${ARG_PUBLIC_URL:-.}
-ENV REACT_APP_BACKEND_URL=${ARG_REACT_APP_BACKEND_URL:-/api}
+ENV VITE_BACKEND_URL=${ARG_VITE_BACKEND_URL:-/api}
```

Comment:
Looking at vite migration made to finesse frontend I think the ouput directory of vite is 'dist': 

https://github.com/ai-cfia/finesse-frontend/blob/8f920c0032c678d944b260ef43ba2dc8e46791a8/Dockerfile#L42

Was the Dockerfile tested locally ?

You might have to change line 25 from 
```
COPY --from=build /nachet-frontend/build ./build
```
to 
```
COPY --from=build /nachet-frontend/dist ./build
```

Reply:
@SonOfLope Thank you!